Userland/Libraries/LibWeb/HTML/Window.cpp

```diff
@@ -1742,6 +1742,9 @@ Window::NamedObjects Window::named_objects(StringView name)
 
 bool Window::find(String const& string)
 {
+    if (m_associated_document)
+        m_associated_document->update_layout();
```

Comment:
Can we move this closer to where the layout state is actually accessed?

Reply:
sure, I did a change in find_matching_text() instead and moved it into a separate commit.